src/commands/commit/prompts.js

```diff
@@ -56,10 +59,10 @@ export default (gitmojis: Array<Gitmoji>, options: CommitOptions): Array<Object>
       name: 'title',
       message: 'Enter the commit title',
       validate: guard.title,
-      transformer: (input: string) => {
-        return `[${
-          (title || input).length
-        }/${TITLE_MAX_LENGTH_COUNT}]: ${input}`
+      transformer: (input: string, { gitmoji }: { gitmoji: string }) => {
+        return `[${(title || input).length}/${
+          TITLE_MAX_LENGTH_COUNT - gitmoji.length
```

Comment:
Hey! If I'm not mistaken, you're not taking into account the space that is being set between the emoji and the commit title, that's why the value was hardcoded to 48 😅 

Also gitmoji.length is not a good way to handle this, because if the user picks the unicode format, the length will be more than 1 but in GitHub the :string: will translate into a single emoji, so I think we can stick with the old approach

Thanks for opening the PR 😊